package.json

```diff
@@ -4,10 +4,13 @@
   "description": "Testing module for lumino node",
   "main": "src/index.js",
   "scripts": {
-    "test": "npx jest --runInBand",
+    "prepare": "npx husky install && scripts/validateResources",
+    "pretest": "scripts/validateResources",
+    "requirements": "scripts/validateResources",
```

Comment:
why is `pretest` the same as `requirements`? when should i use one and not another? and when should i use `prepare`?

Reply:
`pretest` is something that runs automatically when you run `npm test something`, i put that to pre-check requirements before running anything, also we have that on the pre-install hook, this is just to avoid dependency issues when you are trying to run the tests, for example when you switch your branches or things like that. `requirements` it's just there to be used manually if you want to run the validateResources directly.